pages/api/id/[form]/owners.ts

```diff
@@ -0,0 +1,49 @@
+import isRequestAllowed from "../../../../lib/middleware/httpRequestAllowed";
+import { NextApiRequest, NextApiResponse } from "next";
+import dbConnector from "../../../../lib/integration/dbConnector";
+import isUserSessionExist from "../../../../lib/middleware/HttpSessionExist";
+import executeQuery from "../../../../lib/integration/queryManager";
+
+const owners = async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+  try {
+    switch (req.method) {
+      case "GET":
+        return await getEmailListByFormID(req, res);
+      default:
+        res.status(200); // TODO handle card # 491 to associate a specific email to a form
+        break;
+    }
+  } catch (err) {
+    res.status(500).json({ error: `Error on Server Side` });
+  }
+};
+
+export async function getEmailListByFormID(
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> {
+  const formID = req.query.form as string;
+  if (formID) {
+    //Get emails by formID
+    const resultObject = await executeQuery(
+      dbConnector(),
+      "SELECT email FROM form_users WHERE template_id = ($1)",
+      [formID]
+    );
+    type EmailListResponse = { email: string };
+    //Return all emails associated with formID
+    if (resultObject.rowCount > 0)
+      return res.status(200).json(
```

Comment:
This confused me a bit and then I realized its a one line if statement. For readability I think it's okay to add the brackets